src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryMessages.java

```diff
@@ -8,4 +8,9 @@ private MercuryMessages() {}
 
   public static final String LIST_ITEMS_ARE_UNIQUE_MSG = "all list elements are unique";
   public static final String LIST_ITEMS_ARE_NOT_UNIQUE_MSG = "there are at least two elements with the same name";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)